enterprise/frontend/src/embedding-sdk/types/auth-config.ts

```diff
@@ -11,8 +11,20 @@ type BaseMetabaseAuthConfig = {
    * The refresh token should be in the format of {@link UserBackendJwtResponse}
    */
   fetchRequestToken?: MetabaseFetchRequestTokenFn;
+
+  /**
+   * Which authentication method to use.
+   * This should be provided when the instance has more than one configured authentication method.
```

Comment:
nit: I'd be explicit here. Now we only talk about 2 authentication methods JWT and SAML. This gives the impression that if I have Google and JWT enabled then it would default to SAML which doesn't make much sense.

I'd just say if users have both SAML and JWT enabled at the same time it will default to SAML unless `authMethod` is specified.

Reply:
That reads much easier, fixed in https://github.com/metabase/metabase/pull/58641/commits/3544030612bce1b513aa43746dda58a097dbd44a